src/main/java/com/cdancy/bitbucket/rest/fallbacks/BitbucketFallbacks.java

```diff
@@ -663,6 +674,11 @@ public static PullRequest createPullRequestFromErrors(final List<Error> errors)
                 null, null, errors);
     }
 
+    public static SupportZip createSupportZipFromErrors(final List<Error> errors) {
+        return SupportZip.create(null, 0, null,
+                               null, null, null, errors);
```

Comment:
This _shouldn't_ actually work if none of the fields are annotated with @Nullable.